packages/grid/x-data-grid/src/components/toolbar/GridToolbarQuickFilter.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import SearchIcon from '@mui/icons-material/Search';
```

Comment:
Our app started failing to compile this morning because it couldn't resolve `@mui/icons-material`, seems like it is not added in the dependencies of this module. We had to separately install `@mui/icons-material` to get our app working again. We are not using Material Icons in our project at all either, is this an absolute necessity to fetch and use the SearchIcon from Material Icons in this package?

Reply:
This has been reported in #4906
The fix will be available in next release